Tasks/UsePythonVersion/task.json

```diff
@@ -0,0 +1,57 @@
+{
+    "id": "33C63B11-352B-45A2-BA1B-54CB568A29CA",
+    "name": "UsePythonVersion",
+    "friendlyName": "Use Python Version",
+    "description": "Retrieves the specified version of Python from the tool cache. Optionally add it to PATH.",
+    "helpMarkDown": "",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 132,
+        "Patch": 0
+    },
+    "demands": [],
+    "instanceNameFormat": "Use Python $(versionSpec)",
+    "inputs": [
+        {
+            "name": "versionSpec",
+            "type": "string",
+            "label": "Version spec",
+            "required": true,
+            "defaultValue": ">= 3.3",
+            "helpMarkDown": "Version range or exact version of a Python version you want to use."
+        },
+        {
+            "name": "outputVariable",
+            "type": "string",
+            "label": "Output variable",
+            "required": true,
+            "defaultValue": "",
+            "helpMarkDown": "Set a name for the build variable that will contain the directory of the decompressed Python distribution. You can use this in subsequent tasks to access your installation of Python."
```

Comment:
"Set a name for the build variable" --> "Set a name for the variable"
(This could be run in a release in addition to a build.)

"You can use this in subsequent tasks" --> "Use this in subsequent tasks"

"to access your installation of Python." --> "to access the installation of Python."

Reply:
Thanks.  I am going to say "*this* installation of Python" if that sounds good to you.